src/main/java/net/rcarz/jiraclient/User.java

```diff
@@ -111,16 +111,18 @@ public static User get(RestClient restclient, String username)
         return result;
     }
 
-    public static void create(RestClient restClient, String email, String displayName) throws JiraException{
+    public static User create(RestClient restClient, String email, String displayName) throws JiraException{
         JSONObject params = new JSONObject();
         params.put("email", email);
         params.put("displayName", displayName);
 
         try {
             restClient.post("/rest/servicedeskapi/customer", params);
+            return get(restClient, email);
```

Comment:
Can you use the response from the `restClient` to build and return the `User`?

`JSON response = restClient.post("/rest/servicedeskapi/customer", params);
return new User(restclient, (JSONObject) response);`